src/admin-notifications.php

```diff
@@ -56,7 +52,7 @@ public function display_notifications() {
 		echo '<div style="margin: 15px 0 15px -15px;">';
 		foreach ( $notifications as $notification ) {
 			// phpcs:ignore WordPress.XSS.EscapeOutput.OutputNotEscaped
-			echo '<div class="notice notice-' . esc_attr( $notification->get_type() ) . '"><p>' . $notification->get_message() . '</p></div>';
+			echo '<div class="notice notice-' . esc_attr( $notification->get_type() ) . '"><p>' . esc_html( $notification->get_message() ) . '</p></div>';
```

Comment:
Are these messages always without HTML ? Or should `wp_kses_post()` be used here ?